### ISSUE TITLE
Unknown config option "stream_cors_header"

### ISSUE THREAD

**Issue by tbertenshaw** [Motion-Project/motion#1216]
Motion continues to start/run but i can see in the logs the above error. Also get this if i use `*` or `http:\\localhost`

0. Reviewed guide and contributing documents? `(Yes)`:
1. version [x.y.z, hash, other]: 
`Version 4.1.1`
2. installed as a package or compiled from sources [deb, rpm, git, other]: 
`apt-get`
3. standalone or part of third party [motion, MotionEyeOS, other]: 
`motion`
4. video stream source [V4L (card or USB), net cam (mjpeg, rtsp, other), mmal]: 
5. hardware [x86, ARM, other]: 
`Raspberry pi zero`
6. operating system [16.04, Stretch, etc, FreeBSD, other]: 
```
Distributor ID: Raspbian
Description:    Raspbian GNU/Linux 10 (buster)
Release:        10
Codename:       buster
```

##status
```
Sep 28 15:24:21 pi motion[15546]: [9709192:motion] [NTC] [ALL] conf_load: Processing thread 0 - config file /etc/motion/motion.conf
Sep 28 15:24:21 pi motion[15546]: [9709192:motion] [ALR] [ALL] conf_cmdparse: Unknown config option "stream_cors_header"
Sep 28 15:24:21 pi motion[15546]: [9709192:motion] [NTC] [ALL] motion_startup: Motion 4.1.1 Started
Sep 28 15:24:21 pi motion[15546]: [9709192:motion] [NTC] [ALL] motion_startup: Logging to file (/var/log/motion/motion.log)
```

##config
```
############################################################
# Live Stream Server
############################################################

stream_port 8085
stream_quality 50
stream_motion on
stream_maxrate 30
stream_localhost off
stream_limit 0
stream_auth_method 2

stream_authentication user:password
stream_cors_header http://localhost:80
```

**Comment by genius3000** [comment]
That option didn't become available until version 4.2 and newer. The documentation even shows it as non existent in version 4.1.

**Comment by Mr-Dave** [comment]
Thanks g3k, that is correct.  Closing.
